src/uikit.cs

```diff
@@ -5175,6 +5175,164 @@ public interface UIGestureRecognizerDelegate {
+		void ClipToRect (CGRect rect);
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(NSObject))]
+	interface UIGraphicsRenderer
+	{
+		[Export ("initWithBounds:")]
+		IntPtr Constructor (CGRect bounds);
+	
+		[Export ("initWithBounds:format:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGRect bounds, UIGraphicsRendererFormat format);
+	
+		[Export ("format")]
+		UIGraphicsRendererFormat Format { get; }
+	
+		[Export ("allowsImageOutput")]
+		bool AllowsImageOutput { get; }
+	}
+			
+	// Not worth it, Action<UIGraphicsImageRendererContext> conveys more data
+	//delegate void UIGraphicsImageDrawingActions (UIGraphicsImageRendererContext context);
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIGraphicsRendererFormat))]
+	interface UIGraphicsImageRendererFormat
+	{
+		[Export ("scale")]
+		nfloat Scale { get; set; }
+
+		[Export ("opaque")]
+		bool Opaque { get; set; }
+
+		[Export ("prefersExtendedRange")]
+		bool PrefersExtendedRange { get; set; }
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIGraphicsRendererContext))]
+	interface UIGraphicsImageRendererContext
+	{
+		[Export ("currentImage")]
+		UIImage CurrentImage { get; }
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIGraphicsRenderer))]
+	interface UIGraphicsImageRenderer
+	{
+		[Export ("initWithSize:")]
+		IntPtr Constructor (CGSize size);
+
+		[Export ("initWithSize:format:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGSize size, UIGraphicsImageRendererFormat format);
+
+		[Export ("initWithBounds:format:")]
+		[DesignatedInitializer]
+		IntPtr Constructor (CGRect bounds, UIGraphicsImageRendererFormat format);
+
+		[Export ("imageWithActions:")]
+		UIImage CreateImage (Action<UIGraphicsImageRendererContext> actions);
+		
+		[Export ("PNGDataWithActions:")]
+		NSData CreatePng (Action<UIGraphicsImageRendererContext> actions);
+		
+		[Export ("JPEGDataWithCompressionQuality:actions:")]
+		NSData CreateJpeg (nfloat compressionQuality, Action<UIGraphicsImageRendererContext> actions);
+	}
+
+	// Not worth it, Action<UIGraphicsImageRendererContext> conveys more data
+	//delegate void UIGraphicsPdfDrawingActions (UIGraphicsPdfRendererContext context);
+	// Action<UIGraphicsPdfRendererContext>
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIGraphicsRendererFormat))]
+	interface UIGraphicsPdfRendererFormat
+	{
+		[Export ("documentInfo", ArgumentSemantic.Copy)]
+		// TODO: add strongly typed binding
+		NSDictionary<NSString, NSObject> DocumentInfo { get; set; }
+	}
+
+	[iOS (10,0)]
+	[BaseType (typeof(UIGraphicsRendererContext))]
+	interface UIGraphicsPdfRendererContext
+	{
+		[Export ("pdfContextBounds")]
+		CGRect PdfContextBounds { get; }
+
+		[Export ("beginPage")]
+		void BeginPage ();
+
+		[Export ("beginPageWithBounds:pageInfo:")]
+		void BeginPage (CGRect bounds, NSDictionary<NSString, NSObject> pageInfo);
+
+		[Export ("setURL:forRect:")]
+		void SetURL (NSUrl url, CGRect rect);
+
+		[Export ("addDestinationWithName:atPoint:")]
+		void AddDestinationWithName (string name, CGPoint point);
```

Comment:
I would remove the `WithName` suffix (here and next API) as the parameter name provides it.